contrib/babelfishpg_common/src/collation.h

```diff
@@ -83,7 +83,7 @@ typedef struct collation_callbacks
 	/* Function pointers set up by the plugin */
 	char	   *(*EncodingConversion) (const char *s, int len, int src_encoding, int dest_encoding, int *encodedByteLen);
 
-	Oid			(*get_server_collation_oid_internal) (bool missingOk);
+	Oid			(*get_database_or_server_collation_oid_internal) (bool missingOk);
```

Comment:
this function name looks confused to me, since later we also add function like get_database_collation_oid

Reply:
We have updated all such occurrences to get_database_or_server_collation_oid

Reply:
Can we give comments why we have this naming ? Personally I'm confused with this name.

Reply:
Added comment before the function definition in c files.